cfme/tests/integration/test_aws_iam_auth_and_roles.py

```diff
@@ -38,11 +38,12 @@ def pytest_generate_tests(metafunc):
 
 
 @pytest.mark.tier(2)
-@pytest.mark.uncollectif(lambda appliance: appliance.is_dev, reason="Is a rails server")
+@pytest.mark.uncollectif(lambda appliance: appliance.is_dev,
```

Comment:
should this be `temp_appliance`?

Reply:
It shouldn't matter anyway right? Since `temp_appliance` inherits from `appliance` which is used for version/stream defaults.

Reply:
as I see it, it shouldn't matter. but this isn't about version/stream. developers want to run tests locally against their semi-built appliances. if we run tests against remote appliances, it doesn't bring any benefit to them

Reply:
Closing loop here, adding temp_appliance here broke collection.